tests/webview/ui/test_browse.py

```diff
@@ -49,10 +49,22 @@ def test_subject_categories_have_page_and_book_counts(webview_base_url, selenium
     # When the browse link in the navbar is clicked
     browse = home.header.click_search()
 
-    # Then the subject categories have page and book counts
```

Comment:
Please keep the `THEN` comment. 

https://trello.com/c/MljGhE2X/24-test-design-given-when-then